src/charm.py

```diff
@@ -709,22 +729,26 @@ def _get_tracing_endpoint_info(self) -> str:
 
         return self.tracing.otlp_http_endpoint() or ""
 
-    def _update_kratos_endpoints_relation_data(self, event: RelationEvent) -> None:
-        logger.info("Sending endpoints info")
+    def _update_kratos_info_relation_data(self, event: RelationEvent) -> None:
+        logger.info("Sending kratos info")
 
-        admin_endpoint = (
-            self._admin_url
-            or f"http://{self.app.name}.{self.model.name}.svc.cluster.local:{KRATOS_ADMIN_PORT}"
-        )
-        public_endpoint = (
-            self._public_url
-            or f"http://{self.app.name}.{self.model.name}.svc.cluster.local:{KRATOS_PUBLIC_PORT}"
-        )
+        (admin_endpoint, public_endpoint) = self._kratos_endpoints
+        providers_cofigmap_name = "providers"
+        schemas_cofigmap_name = "identity-schemas"
```

Comment:
Let's not hardcode this
```suggestion
        providers_configmap_name = self.providers_configmap.name
        schemas_configmap_name = self.schemas_configmap.name
```